gulpfile.js

```diff
@@ -10,6 +10,13 @@ gulp.task("govuk-frontend-copy", function() {
   ]).pipe(gulp.dest("./dist/static"));
 });
 
+// copies the all js from the govuk-frontend package to the dist directory
```

Comment:
not sure this line makes sense - remove either the or all

Reply:
updated, thanks